sql_joins/sql_joins.md

```diff
@@ -0,0 +1,875 @@
+[[ ]] Join criteria are singular, and you can only indicate one relationship that makes data "match".
+[[ ]] Join criteria include "left", "right", "inner", and "full".
+[[X]] Join criteria can involve different kinds of relationships, like equality or one field being between two other fields.
+[[?]] There is more than one correct answer!
+****
+<div class = "answer">
+
+Join criteria are statements about relationships between data elements, such as fields being equal, one field being less than another, one field being between two other fields, and so on.
+
+You can use multiple conditions to describe data relationships, such as two fields matching exactly, or one field being equal while another field is less than a third field.  You can certainly include more than one relationship that makes data match, and you'll often have to!
+
+The term for indicating which table's data should be included and according to what part of a two-table overlap of data is "join type".  Join types include "left", "right", "inner", and "full", but these terms don't apply to "join criteria".
+
+</div>
+****
+
+### Combining Join Type and Join Criteria
+
+In the next few sections, we'll combine our join type and join criteria, to show you how these work together.
+
+You'll see a `FROM` statement, which describes the type of join, and an `ON` statement, which describes the join criteria.
+
+Importantly, we will only show a few examples.  There are many combinations we could consider: 
+
+* A `LEFT JOIN` involving tables which require a "between" type relationship (like a "spend" value between a "budget\_floor" value and a "budget\_ceiling" value)
+* An `INNER JOIN` involving equality matching on three fields: two identifiers and one date field
+* A `RIGHT JOIN` involving matching data with a "less than" relationship, such as a 3 month old weight measurement being less than a 1 month old weight measurement
+* and many more!
+
+Instead of being exhaustive, we've concentrated on the most frequent use case you'll encounter over and over: equality.  We'll go over each join type (inner, left, right, and full) on a simple equality matching a single field from each table.
+
+#### `INNER JOIN` and Equality Condition
+
+To understand what an `INNER JOIN` with equality looks and acts like practically, let's go to a simple example of two tables we used earlier in this module:
+
+**disease**
+
+<!-- data-type="none" class="tight-table"-->
+| subject\_id  | lung\_cancer |
+| :--------- | :--------- | 
+| 3  | TRUE |
+| 5 | FALSE  |
+| 8  | FALSE |
+
+**smoking**
+
+<!-- data-type="none" class="tight-table"-->
+| subject\_id  | smoking\_pack\_years |
+| :--------- | :--------- | 
+| 2  | 10 |
+| 3 | 10  |
+| 4  | 0 |
+
+And let's combine our join criteria (subject\_id matching) with our join type (inner).  We won't show the whole query here, just the pertinent parts that describe the join:
```

Comment:
Would you consider having the first line of the code be `SELECT *` instead of `...`?
Given that the entirety of the code seems no longer than the abbreviated code, and including `SELECT *` doesn't seem that distracting. I'm personally more distracted that it isn't there.